src/components/CreateCourseRunPage/index.jsx

```diff
@@ -129,8 +129,9 @@ class CreateCourseRunPage extends React.Component {
     const showSpinner = !startedFetching || courseInfo.isFetching || courseRunOptions.isFetching;
     const showForm = startedFetching && !courseInfo.isFetching && !courseRunOptions.isFetching &&
       !courseInReview;
-    const sortedRunLabels = course_runs && this.parseCourseRunLabels(course_runs);
-    const defaultRun = sortedRunLabels ? sortedRunLabels[0].value : '';
+    const sortedRunLabels = (showForm &&
```

Comment:
Found a bug where the select field on the 'add course run' page will cause an error for courses without existing runs. Wanted to push a fix for this edge case out with these edits prior to bug bash.